css-two-column-layout/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="header">
+    <h1>Scale, Speed, Agility</h1>
+    <p class="header">We've Dedicated thousands of man hours to making healthcare institutions in the Helium network run 10x more efficiently and deliver an unparalleled quality of care.</p>
+  </div>
+  <div class="container">
+    <div class="row">
+      <div class="one-half">
+        <div class="one-fifth">
+          <img src="images/software.svg" alt="software">
+        </div>
+        <div class="four-fifths">
+          <section>
+            <h3>intuitive, SMART SOFTWARE</h3>
+            <p>We build products that transform how healthcare facilities run across Africa- making them radically more efficient operationally and financially, a complete turnaround.</p>
+          </section>
+        </div>
+      </div class="one-half">
+      <div class="one-half">
+        <div class="one-fifth">
+          <img src="images/practice.svg" alt="stethoscope">
+        </div>
+        <div class="four-fifths">
+          <section>
+            <h3>Practice Medicine, not administration</h3>
+            <p>Our all-in-one electronic health records (EHR), hospital management (HMIS) &amp; billing system helps streamline your facility, reduce waste and improve quality of care.</p>
+          </section>
+        </div>
+      </div class="one-half">
+    </div class="row">
+    <div class="row">
+      <div class="one-half">
+        <div class="one-fifth">
+          <img src="images/circumstance.svg" alt="atoms">
+        </div>
+        <div class="four-fifths">
+          <section>
+            <h3>For any circumstance</h3>
+            <p>Our intuitive system is built around the peculiarities of your facility; we support offline deployments, sync to the cloud and run on any device 24/7.</p>
+          </section>
+        </div>
+      </div class="one-half">
+      <div class="one-half">
+        <div class="one-fifth">
```

Comment:
make sure to include a `one-fifth` class in the CSS with a proper flex basis assigned to it.